src/stack_mitosis/interpreter.clj

```diff
@@ -118,15 +143,15 @@
   (sudo/sudo-provider (sudo/load-role "resources/role.edn"))
   (def rds (client))
   (-> (predict/state [] (example/create example/template))
-      (plan/replace-tree "mitosis-prod" "mitosis-demo"))
+      (plan/replace-tree "mitosis-prod" "mitosis-demo" nil))
```

Comment:
It looks like the argument for missing snapshot has changed from `nil` to `:none`? I know these are in comment blocks, but might be better to be consistent. My preference would be to keep it as `nil` and change the logic calculating a snapshot to return nil on nothing found. I'm guessing the `:none` placeholder was put in to help guard against invalid or missing values, but as it currently stands this example will fall through to the snapshot case (as nil != :none), and then fail because it's an invalid snapshot. 

(the :none check appears to be https://github.com/dgtized/stack-mitosis/pull/7/files#diff-6d5b0062041657017d820d3bafeecad3300531e355a3adb93bc26694a9b17dfdR45)

Reply:
Fixed!